lib/src/widgets/currency_conversion_text_field.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+import '../utils/custom_number_format.dart';
+import './text_with_border.dart';
+
+class CurrencyConverSionTextField extends StatelessWidget {
+  const CurrencyConverSionTextField({
+    Key? key,
+    required this.labelText,
+    this.isCrypto = true,
+    this.onChanged,
+    this.currentCryptoPrice,
+  }) : super(key: key);
+
+  final String labelText;
+  final bool isCrypto;
+  final ValueChanged<String>? onChanged;
+  final double? currentCryptoPrice;
+
+  @override
+  Widget build(BuildContext context) {
+    return TextField(
+      onChanged: (value) => onChanged!(value),
```

Comment:
what happens if onChanged null

Reply:
When the user enter number in  TextField, I can get [value] of TextField and convert it. Ex : 
- Enter 1 in BTC TexField => display $62,614.00
- Enter 2 in BTC TextField => display $125,228.00

Reply:
I just consider if the value be null, your app may be crash